Makefile

```diff
@@ -14,6 +14,11 @@ WAIT_GRAYLOG = $(COMPOSE_RUN) dockerize -wait tcp://graylog:9000 -timeout 120s
 default: help
 
 # -- Project
+package-lock.json: package.json
+	@echo "Updating package-lock.json file..."
+	@$(COMPOSE_RUN) -T --no-deps etherpad \
+	  cat package-lock.json > package-lock.json
```

Comment:
this is the first time I see this :grimacing: it's a little bit strange... but why not.

Reply:
How would you handle it? This avoids having to build locally (outside from Docker) to update the file.

Reply:
I don't know, this looks like a little bit shaky and error prone. It's not possible to use this project without this makefile

Reply:
Can you explain how do you proceed to update Marsha's project lock file?

Reply:
in richie or marsha the lock file is updated when we add a library or by the renovate bot.

Reply:
Current rule avoid having to run dependency updates locally. I'm not sure to understand why you think it would be error prone.